docs/samples/Local TXDC Setup.md

```diff
@@ -97,10 +96,9 @@ helm install sokrates charts/tractusx-connector  \
     --namespace cx \
     --create-namespace \
     --set fullnameOverride=sokrates \
-    --set controlplane.image.tag=0.2.0 \
+    --set controlplane.image.tag=latest \
```

Comment:
using `latest` is dangerous, because the `latest` tag gets overwritten on every build we do on tractusx-edc